index.html

```diff
@@ -171,7 +189,7 @@
             });
             card.appendChild(deleteButton);
 
-            app.appendChild(card);
+            cards.appendChild(card);
```

Comment:
Replace the existing app.appendChild(card); line in the createCard function with: